artillery/artillery-data.js

```diff
@@ -0,0 +1,14 @@
+const generateRandomId = (userContext, events, done) => {
+  const min = 5000000;
+  const max = 10000000;
+
+  const id = Math.floor(Math.random() * ((max - min) + 1)) + min;
```

Comment:
You can use Math.ceil() instead of Math.floor() and get rid of the + 1 I believe.